### PR TITLE
DockerCheatSheet : updated according to the official docs

### LINKED ISSUE #3694: Improvement in Docker Cheat Sheet
Currently Docker cheat sheet uses information from unofficial resources, also it lacks some information. Idea is to use information from the official site itself.  

Official command line reference: [Docker docs](https://docs.docker.com/engine/reference/commandline/)
IA Page: https://duck.co/ia/view/docker_cheat_sheet 

cc @moollaza, @GuiltyDolphin

### PR DESCRIPTION
**Pull request by prabhakar267**
fixes #3694 
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`DockerCheatSheet : Fix 3694`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

What changes does this PR introduce? 

###### Related Issues or Discussions 

What issues or discussions (if any) does this PR address? Please provide the issue # or link.
#3694 

###### People to notify

@mention any relevant people/organizations
@sahildua2305 

---

Instant Answer Page: https://duck.co/ia/view/docker_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @preemeijer

### CODE REVIEW

**Comment by daxtheduck** [comment]
@prabhakar267 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/docker.json [hunk 1]
```diff
@@ -8,221 +8,313 @@
         "sourceUrl": "https://github.com/wsargent/docker-cheat-sheet"
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/docker.json]: @prabhakar267 can you please update the new source (name and URL) as well?

**Comment by prabhakar267** [comment]
@sahildua2305  I made the changes in meta data

**Comment by preemeijer** [comment]
Hello @prabhakar267,

Some nice changes... The "original" cheatsheet is somewhat old. Things have changed.
If people are happy with the new one no problem on my side. 

Side note: Docker is getting easier to use! :+1: 

Cheers, Paul

**Comment by gautamkrishnar** [comment]
@prabhakar267 Hurray! 🎉  its now merged... Thanks for taking your time to contribute...

**Pull request merged**